pkg/archive/copy_test.go

```diff
@@ -310,11 +337,21 @@ func TestCopyCaseB(t *testing.T) {
 	if err != ErrDirNotExists {
 		t.Fatalf("expected ErrDirNotExists error, but got %T: %s", err, err)
 	}
+
+	symlinkPath := filepath.Join(tmpDirA, "symlink3")
+
+	if err = testCopyHelperFSym(t, symlinkPath, dstDir); err == nil {
+		t.Fatal("expected ErrDirNotExists error, but got nil instead")
+	}
+	if err != ErrDirNotExists {
+		t.Fatalf("expected ErrDirNotExists error, but got %T: %s", err, err)
+	}
+
 }
 
-// C. SRC specifies a file and DST exists as a file. This should overwrite
-//    the file at DST with the contents of the source file.
-func TestCopyCaseC(t *testing.T) {
+// C.1 SRC specifies a file and DST exists as a file. This should overwrite
+//     the file at DST with the contents of the source file.
+func TestCopyCaseC1(t *testing.T) {
```

Comment:
I'd use `TestCopyCaseCFollowSymlink` or `TestCopyCaseCFSym` instead the `1/2` suffix.